### ISSUE TITLE
COLAB preinstalled CUDA toolkit version is 10.1

### ISSUE THREAD

**Issue by yhgon** [rapidsai/rapidsai-csp-utils#25]
when check  the version of cudatoolkit,  pre-installed version is 10.1 
```
Python 3.6.9
Cuda compilation tools, release 10.1, V10.1.243
```
conda install support 10.1 but csp utils script provide  only 10.0 for stable and nightly build. 
In the environment, rapids works well.  However, when I try to integrate rapids to other legacy tools such as py-openmm, it have problem to load module.  It need to detect pre-installed  CUDA  toolkit version and select proper one.

**Comment by yhgon** [comment]
I found  v0.13 support  CUDA toolkit v10.1

**Comment by taureandyernv** [comment]
on it!  thanks!

**Comment by taureandyernv** [comment]
Closed as fixed by PR #26
